upup/pkg/fi/cloudup/utils.go

```diff
@@ -161,6 +162,21 @@ func BuildCloud(cluster *kops.Cluster) (fi.Cloud, error) {
 
 			cloud = azureCloud
 		}
+	case kops.CloudProviderScaleway:
+		{
+			region, zone, err := scaleway.FindRegionAndZone(cluster)
+			if err != nil {
+				return nil, err
+			}
+
+			// TODO: handle tags
```

Comment:
Are tags mandatory in a first version? If not let's remove this comment for the time being.